adapters/rubicon/rubicontest/exemplary/app-imp-fpd.json

```diff
@@ -141,21 +141,13 @@
           "h": 576
         },
         "ext": {
-          "context": {
-            "data": {
-              "adserver": {
-                "name": "gam",
-                "adslot": "adSlotFromContextData"
-              },
-              "dataAttr1": "dataVal1",
-              "dataAttr2": "dataVal2"
-            }
-          },
           "data": {
             "adserver": {
               "name": "gam",
-              "adslot": "adSlotFromData"
-            }
+              "adslot": "adSlotFromContextData"
```

Comment:
Nitpick: would it be more appropriate if the references to `"adSlotFromContextData"` in this file were `"adSlotFromData"`?